src/client/pages/RetroCertsLandingPage/index.js

```diff
@@ -11,12 +12,12 @@ function RetroCertsLandingPage(props) {
   const history = useHistory();
 
   if (!userData.weeksToCertify) {
-    const authToken = sessionStorage.getItem("authToken");
+    const authToken = sessionStorage.getItem(auth.AUTHTOKEN);
```

Comment:
minor, is there an easy way to make it clearer that auth just contains a lot of constant strings/slugs? that AUTHTOKEN is "authToken"

Reply:
(see comment in auth.js)